kitsune/search/tests/signals/test_forums.py

```diff
@@ -7,6 +7,7 @@
 
 class ForumDocumentSignalsTests(Elastic7TestCase):
     def setUp(self):
+        super().setUp()
```

Comment:
I am not sure that we need this

Reply:
Since I changed the `ElasticTestCase` it is needed. Now we set the test environment up cleanly, force a refresh, and get in front of any index issues.